### PR TITLE
Update payload values for API to include configuration

### PR DESCRIPTION
**Pull request by jayco**
We had a customer let us know that the configuration value was not mentioned in the required payload, this updates that value in two places.

Create pipeline:
Screen Shot 2020-12-04 at 9 27 55 am

Update pipeline:
Screen Shot 2020-12-04 at 9 27 46 am

### CODE REVIEW

**Comment by jayco** [comment]
Forget that, those values are for parameter responses

**Comment by jayco** [comment]
Also removing this confusing heading "Parameters responses for both YAML pipelines and Visual step pipelines" which is placed above the Required request params

**Review by plaindocs — COMMENTED** [review]
If we're sure those work for both, I'm happy to just merge this and improve later.

File: pages/apis/rest_api/pipelines.md.erb [hunk 1]
```diff
@@ -330,21 +328,9 @@ Required [request body properties](/docs/api#request-body-properties):
...
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/wait-step">Wait</a> for all previous steps to finish:</em> <code>{ "type": "waiter" }</code>
-      </p>
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/block-step">Block</a> pipeline (see the <a href="/docs/apis/rest-api/jobs#unblock-a-job">job unblock API</a>):</em> <code>{ "type": "manual" }</code>
-      </p>
-      <p class="Docs__api-param-eg">
-        <em><a href="/docs/pipelines/trigger-step">Trigger</a> pipeline:</em> <code>{ "type": "trigger", "trigger_project_slug": "a-pipeline" }</code>
-      </p>
+      The YAML pipeline that consists of the build pipeline steps.<p class="Docs__api-param-eg"><em>Example:</em> <code>"steps:\n - command: \"script/release.sh\"\n "Build \:package\:"\n""</code>
```
Comment (plaindocs) [inline pages/apis/rest_api/pipelines.md.erb:333]: This fails for me with

```
{
  "message": "Problems parsing JSON"
}
```

Comment (plaindocs) [reply pages/apis/rest_api/pipelines.md.erb:333]: -d '{
    "name": "My Pipeline",
    "repository": "git@github.com:acme-inc/my-pipeline.git",
    "configuration": "steps:\n - command: \"script/release.sh\"\n " 
  }'

Does work

File: pages/apis/rest_api/pipelines.md.erb [hunk 2]
```diff
@@ -315,8 +315,6 @@ The resulting pipeline:
 
 <%= image 'pipeline-example.png', alt: 'Image of the pipeline steps that are created in the Buildkite UI', width: 974/2, height: 90/2 %>
 
-### Parameters responses for both YAML pipelines and Visual step pipelines
```
[the remark this answers is not in the document]
Comment (jayco) [reply pages/apis/rest_api/pipelines.md.erb:318]: let's catch up and chat about it :)

File: pages/apis/rest_api/pipelines.md.erb (hunk 1 above)
Comment (jayco) [reply pages/apis/rest_api/pipelines.md.erb:333]: Same as above, let's have a catch-up and chat about it

File: pages/apis/rest_api/pipelines.md.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jayco) [reply pages/apis/rest_api/pipelines.md.erb:318]: Yeah, so the reason I removed that is that it sits above the **request** body properties, and is confusing as it the heading for the responses:

Screen Shot 2020-12-11 at 12 33 45 pm

File: pages/apis/rest_api/pipelines.md.erb (hunk 1 above)
Comment (jayco) [reply pages/apis/rest_api/pipelines.md.erb]: Good pick up, fixed

**Review by plaindocs — APPROVED** [review]

**Pull request merged**